src/main/java/org/gridsuite/directory/server/dto/filter/AbstractFilter.java

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
You should not copy classes from filter-server inside the directory-server, its code duplication and its risky.
If you wathc the contingency dev for their integration in this directory-server, the content pass throught  the directory server to be sent to the concerned micro service which is responsible to check its consistency.
Then you should not have `AbstractFilter` or other `*Filter` objects in here. Simply pass the json content I think

Reply:
deleted